Makefile

```diff
@@ -71,25 +75,53 @@ lit:
 pytest:
 	@echo "check the Catalyst PyTest suite"
 ifdef remotetests
-	$(PYTHON) pytest frontend/test/pytest --tb=native --backend=$(TEST_BACKEND) --runbraket=$(TEST_BRAKET) -n auto
+	$(PYTHON) -m pytest frontend/test/pytest --tb=native --backend=$(TEST_BACKEND) --runbraket=$(TEST_BRAKET) -n auto
 else
-	$(PYTHON) pytest frontend/test/pytest --tb=native --backend=$(TEST_BACKEND) --runbraket=$(TEST_BRAKET) -k "not remotetests" -n auto
+	$(PYTHON) -m pytest frontend/test/pytest --tb=native --backend=$(TEST_BACKEND) --runbraket=$(TEST_BRAKET) -k "not remotetests" -n auto
 endif
 test-demos:
 	@echo "check the Catalyst demos"
 	MDD_BENCHMARK_PRECISION=1 \
-	$(PYTHON) pytest demos/*.ipynb --nbmake -n auto
+	$(PYTHON) -m pytest demos/*.ipynb --nbmake -n auto
+
+wheel:
+	echo "INSTALLED = True" > $(MK_DIR)/frontend/catalyst/_configuration.py
+	# Copy bins to frontend/catalyst/bin
+	mkdir -p $(MK_DIR)/frontend/catalyst/bin
+	cp $(LLVM_BUILD_DIR)/bin/llc $(MK_DIR)/frontend/catalyst/bin
+	cp $(LLVM_BUILD_DIR)/bin/opt $(MK_DIR)/frontend/catalyst/bin
+	cp $(LLVM_BUILD_DIR)/bin/mlir-translate $(MK_DIR)/frontend/catalyst/bin
+	cp $(MHLO_BUILD_DIR)/bin/mlir-hlo-opt $(MK_DIR)/frontend/catalyst/bin
+	cp $(DIALECTS_BUILD_DIR)/bin/quantum-opt $(MK_DIR)/frontend/catalyst/bin
+	# Copy libs to frontend/catalyst/lib
+	mkdir -p $(MK_DIR)/frontend/catalyst/lib/backend/ $(MK_DIR)/frontend/catalyst/lib/capi
+	cp $(RT_BUILD_DIR)/lib/backend/librt_backend.so $(MK_DIR)/frontend/catalyst/lib/backend/
+	cp $(RT_BUILD_DIR)/lib/capi/librt_capi.so $(MK_DIR)/frontend/catalyst/lib/capi
+	cp --dereference $(LLVM_BUILD_DIR)/lib/libmlir_float16_utils.so.* $(MK_DIR)/frontend/catalyst/lib
+	cp --dereference $(LLVM_BUILD_DIR)/lib/libmlir_c_runner_utils.so* $(MK_DIR)/frontend/catalyst/lib
+	# Copy enzyme to frontend
+	cp --dereference $(ENZYME_BUILD_DIR)/Enzyme/LLVMEnzyme-17.so $(MK_DIR)/frontend/catalyst/lib
+	# Copy mlir bindings to frontend/mlir_quantum
+	mkdir -p $(MK_DIR)/frontend/mlir_quantum/dialects
+	cp -R --dereference $(DIALECTS_BUILD_DIR)/python_packages/quantum/mlir_quantum/runtime $(MK_DIR)/frontend/mlir_quantum/runtime
+	cp --dereference $(DIALECTS_BUILD_DIR)/python_packages/quantum/mlir_quantum/ir.py $(MK_DIR)/frontend/mlir_quantum/
+	for file in arith tensor scf gradient quantum _ods_common ; do \
+		cp --dereference $(DIALECTS_BUILD_DIR)/python_packages/quantum/mlir_quantum/dialects/*$${file}* $(MK_DIR)/frontend/mlir_quantum/dialects ; \
+	done
+	find $(MK_DIR)/frontend -type d -name __pycache__ -exec rm -rf {} +
```

Comment:
I have concerns about the compatibility and effort required to use these commands with future work on Windows (and MacOS). But, let's revisit this when we have a better support of building C++ modules from Python.

Reply:
Sure, but those are the commands we have right now 😅

The fact that the build recipe for wheels has been extracted to a central location is a good thing, and it doesn't prevent whatever changes you want to make in the future :)